packages/charts/src/Charts/SeverityLine/SeverityLine.js

```diff
@@ -0,0 +1,77 @@
+import './severity-line.scss';
+
+import React, { Component } from 'react';
+
+import PropTypes from 'prop-types';
+
+class SeverityLine extends Component {
```

Comment:
I'd prefer function component for this, everything should be doable with `useState` hook.